### PR TITLE
[Docs] Markdown hyperlink formatting and the use of the triple backslash for easy command copy.

### PR DESCRIPTION
**Pull request by Abhinavcode13**
- Updated README with clear installation steps for Vagrant and VirtualBox, emphasizing system requirements and user account details for accessing the virtual machine.

### CODE REVIEW

**Comment by Abhinavcode13** [comment]
@jafingerhut Could you please review this PR?
Thank you

File: README.md [hunk 1]
```diff
@@ -74,18 +74,38 @@ then we've already provided you with a virtual machine that has all of
...
+   vagrant up
+   ```
+   *Note* : The time for this step depends on your computer and Internet speed. On a 2015 MacBook Pro with a 50 Mbps download speed, it took approximately 20 minutes. Ensure a stable Internet connection throughout the process.
+
+### Accessing the VM
+
+
+- There are two user accounts:
+  - Username: vagrant | Password: vagrant (This is the default account)
+  - Username: p4 | Password: p4 (This account is expected to use)
```
Comment (jafingerhut) [inline README.md:108]: Please use a complete English sentence instead of "This account is expected to use", which is incomplete.

**Comment by Abhinavcode13** [comment]
Made the modifications that were required. @jafingerhut

**Review by jafingerhut — APPROVED** [review]
LGTM

**Pull request merged**